sklearn/metrics/tests/test_classification.py

```diff
@@ -135,7 +134,7 @@ def test_classification_report_dictionary_output():
         target_names=iris.target_names, output_dict=True)
 
     # assert the 2 dicts are equal.
-    assert(report.keys() == expected_report.keys())
+    assert (report.keys() == expected_report.keys())
```

Comment:
There are some unrelated changes in this PR, most likely caused by an editor. May these changes be reverted?

Reply:
I check it.

Reply:
Indeed, I didn't notice htose changes in a commit.